<launch>
	<node pkg="leader_election" type="leader_election_node" name="A" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+10+150 -e ">
		<param name="UID" value="1" />
		<param name="child1" value="3" />
		<param name="child2" value="4" />
		<param name="parent1" value="3" />
		<param name="parent2" value="2" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
	<node pkg="leader_election" type="leader_election_node" name="B" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+1400+100 -e ">
		<param name="UID" value="2" />
		<param name="child1" value="1" />
		<param name="child2" value="4" />
		<param name="child3" value="5" />
		<param name="parent1" value="5" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
	<node pkg="leader_election" type="leader_election_node" name="C" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+2800+100 -e ">
		<param name="UID" value="3" />
		<param name="child1" value="1" />
		<param name="parent1" value="1" />
		<param name="parent2" value="6" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
	<node pkg="leader_election" type="leader_election_node" name="D" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+600+650 -e ">
		<param name="UID" value="4" />
		<param name="child1" value="5" />
		<param name="parent1" value="1" />
		<param name="parent2" value="2" />
		<param name="parent3" value="6" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
	<node pkg="leader_election" type="leader_election_node" name="E" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+1400+650 -e ">
		<param name="UID" value="5" />
		<param name="child1" value="2" />
		<param name="child2" value="6" />
		<param name="parent1" value="2" />
		<param name="parent2" value="4" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
	<node pkg="leader_election" type="leader_election_node" name="F" output="screen" launch-prefix="xterm -bg Teal -geometry 150x31+2800+650 -e ">
		<param name="UID" value="6" />
		<param name="child1" value="3" />
		<param name="child2" value="4" />
		<param name="parent1" value="5" />
		<param name="diam" value="4" />
		<param name="round" value="1" />
	</node>
</launch>